src/Cli/dotnet/Commands/Tool/Install/ToolInstallLocalCommand.cs

```diff
@@ -194,18 +194,8 @@ public int InstallNewTool(FilePath manifestFile, PackageId packageId)
 
     public FilePath GetManifestFilePath()
     {
-        try
-        {
-            return string.IsNullOrWhiteSpace(_explicitManifestFile)
-                ? _toolManifestFinder.FindFirst(_createManifestIfNeeded)
-                : new FilePath(_explicitManifestFile);
-        }
-        catch (ToolManifestCannotBeFoundException e)
```

Comment:
Why don't we want this to be graceful?

Reply:
It throws `ToolManifestCannotBeFoundException` in `findFirst`, which is a child class of `GracefulException`